README.md

```diff
@@ -428,6 +428,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [InspirAda na Computação](https://inspiradanacomputacao.com/) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
   
     Collaborative blog to give voice to women in computing, science and technology. The blog aims to share content in order to inspire women to become interested in the tech area. It promotes the access to the tech knowledge and brings, as a result, the equality of opportunities.
+    
+  - [OxenTIMenina](https://www.facebook.com/OxenTIMenina/) 
+  
+    Initiative that promotes technology, education and inspire girls in TI.
```

Comment:
1. 'that' should be thats
2. What is TI, is it IT or something else, I don't think the readers will know over here.

In general this does not say much, can you elaborate, do they run workshops, help with internships, provide learning materials, run awareness programs?

Reply:
Thanks for explaining :)